HW03/veaceslav_irizanov/veaceslav_irizanov_hw_03_t_02.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Homework3
+  CONDITION_ERROR = 'ERROR'
+  CONDITION = %r{\S+\s-\s-\s\D\d+/\w+/\S+\s\+\S+\s\S+\s\S+\s\w+/\d\.\d"\s\d+\s\S+\s\S+}
+  DATETIME = %r{\d{2}/\w{3}/\d{4}:\d{2}:\d{2}:\d{2}\s\+\d{4}}
+  IP = /\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/
+  ADRESS = %r{/\w*/\d*/\w*}
+  def task2(log)
+    rows = log.split("\n").delete_if { |str| str.include?(CONDITION_ERROR) }
```

Comment:
`an array of formatted strings containing information about post requests;`

you should take lines with POST request, so u shouldn't delete lines just with ERROR msg